docs/catalog/cloud/reduce-network-traversal-between-VMs.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- cloud
+- network
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Reduce network traversal between VMs
+
+## Description
+Placing VMs in a single region or a single availability zone reduces the physical distance between the instances. 
+
+However, for business critical workloads, you need to ensure your workload is spread across multiple availability-zones, which may result in more network traversal and increase in your carbon footprint.
+
+## Solution
+Consider using VM-Host affinitry rules (also called proximity placement groups) to reduce the network traversal by ensuring your VM resources are physically located close to each other. 
```

Comment:
Please, fix the typo - affinity.

Reply:
I would argue that this could be formulated broader, maybe: choose the VM placement that is best for your workload. Also it would be good to be a bit more vendor agnostic.

Most cloud providers provide various options, like https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html , https://cloud.google.com/compute/docs/instances/define-instance-placement , (Azure you already mentioned) .

Some of these placement groups could be a good trade-off between availability and energy-usage.